acp/admin.php

```diff
@@ -34,17 +34,17 @@
 require JAXBOARDS_ROOT . '/acp/page.php';
 
 /**
- * Strip slashes from input, recurisvely.
+ * Strip slashes from input, recursively.
  *
  * @param mixed $input The input to strip slashes from
  *
  * @return mixed The input, without slashes
  */
 function recursiveStripSlashes($input)
 {
-    if (!get_magic_quotes_gpc()) {
+    /*if (!get_magic_quotes_gpc()) {
         return $input;
-    }
+    }*/
```

Comment:
I believe PHP killed magic quotes in one of the versions between 4 and 8 (GOOD riddance). We can remove this now.